applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.h

```diff
@@ -23,6 +23,7 @@
 #include "compressible_potential_flow_application_variables.h"
 #include "utilities/geometry_utilities.h"
 #include "utilities/enrichment_utilities.h"
+#include "includes/cfd_variables.h"
```

Comment:
I think that this can go to the cpp.

Reply:
Yes it can. Just for me to understand, what is the reason for moving the variable includes to the cpp?

Reply:
if you have includes in the header then if this header is included somewhere else then the includes will also be visible there. I think this can increase the compilation time and theoretically also cause name-clashes if names are identical